roles/cs.versionlock/templates/versionlock.list.j2

```diff
@@ -2,6 +2,10 @@
 {{ pkg }}
 {% endfor %}
 
+{% if mageops_elasticsearch_version is defined %}
```

Comment:
in that way it'll be defined for every node, please template it inside all.yaml as discussed over slack